service/src/ocall_bridge/bridge_api.rs

```diff
@@ -174,11 +174,9 @@ pub trait RemoteAttestationBridge {
 pub trait WorkerOnChainBridge {
 	fn worker_request(&self, request: Vec<u8>) -> OCallBridgeResult<Vec<u8>>;
 
-	fn send_block_and_confirmation(
-		&self,
-		confirmations: Vec<u8>,
-		signed_blocks: Vec<u8>,
-	) -> OCallBridgeResult<()>;
+	fn send_sidechain_blocks(&self, signed_blocks: Vec<u8>) -> OCallBridgeResult<()>;
```

Comment:
Renaming suggestion (@clangenb may correct me here): `propose_sidechain_blocks`

Reply:
Yes, I agree, and I think we should actually move the function from another place. The current location was confusing for me. This function gossips the sidechain blocks to peer-validateers now, it has nothing to do with `WorkerOnchainBridge`.

However, I see now that cleanly separating the sidechain stuff means that we kind of need to replicate some structure from the core-primitives domain into the sidechain domain. This might also be an overkill, so we might just make an extra trait here called `SidechainBridge` or so?

Finally, @haerdib I think you need to become consistent with your sidechain-block naming. In the teerex pallet, you introduced the calls like `propose_sidechainblock`. ;)

Reply:
I will do the re-naming and create a new bridge, called `SideChainBridge`, to where I'll move that function.

Reply:
Oh, you're right @clangenb . Sorry for that - `sidechainblocks` it is ;)

Reply:
TBH I don't really like `propose_sidechainblocks`, it's just not how you would write it in English. `Sidechain` can be a composed noun, but `sidechainblock` is more like you would form the word in German than in English (where I'd prefer `sidechain block`)

Reply:
I can agree with both, as long as we stay consistent, I guess.

Reply:
Well, I guess you could argue that we are in an emerging field, and we henceforth coin the term 'Sidechainblock`, I don't have a strong preference here, but except for the pallet, we have been going with 'sidechain block' until now.

Reply:
Lets go with `_block` then. I'll adapt the pallet.

Reply:
For `sidechain` and `parentchain` I can agree to have it as a compound noun. But I believe it's very rare in English to have a compound noun made of three words.